docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -155,102 +158,412 @@
     "print(\"Successful authentication to Google Cloud.\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "1Oc5MnAWSEkH"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Quantum Engine Configuration\n",
+    "# @markdown Replace the `PROCESSOR_ID` and `YOUR_PROJECT_ID` with your details.\n",
+    "PROCESSOR_ID = 'weber' #@param {type: 'string'}\n",
+    "cals = cirq.google.get_engine_calibration(PROCESSOR_ID, YOUR_PROJECT_ID)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "QCszNh2dTW_O"
+   },
+   "outputs": [],
+   "source": [
+    "#@title View available calibration metrics\n",
+    "#@markdown See https://quantumai.google/cirq/google/calibration for a description of the available calibration metrics\n",
+    "for k, v in cals.items():\n",
+    "  print(k)"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "f97d4bc65b8d"
+    "id": "CXo7Amk_9xUU"
    },
    "source": [
-    "## Heatmaps for Calibration Metrics\n",
-    "\n",
-    "Calibration Metrics about the performance of the quantum processor can be retrieved from the Google QCS."
+    "# Metrics"
```

Comment:
I think we should give at least a short explanation for each of the sections below and how to use them.

Reply:
The sections below just call `plot` function for each of the different calibration metric keys. Should I try to add a description of why each of the charts is useful and what information it provides us?

Reply:
Since the [calibration metrics doc](https://quantumai.google/cirq/google/calibration) provides a full definition for each of these, here's my suggestion:
- For each major (`##`) heading, copy a short (<= 1 sentence) description of that metric from the metrics doc and paste it under the major heading. For example:
    ```
    ## Isolated Pair XEB, sqrt(iswap)
    Two qubit error is primarily characterized by applying cross-entropy benchmarking (XEB).
    ```
- Immediately under the `Metrics` header, link to the metrics doc with a note that it provides more complete definitions of the fields plotted below.

This increases the value of this doc in isolation (less tabbing back-and-forth with the metrics doc) while providing additional details if/when they are needed.

Reply:
done.